.github/workflows/tics.yml

```diff
@@ -17,6 +17,9 @@ concurrency:
 
 jobs:
   TICS:
+    # Only run if we have access to secrets.
+    if: ${{ github.event_name != 'pull_request' || github.repository == github.event.pull_request.head.repo.full_name }}
```

Comment:
The outer `${{ ... }}` shouldn't be needed here.

I wonder if it could be simplified further to just check the secret directly? Something like:

    if: secrets.TICSAUTHTOKEN != ''

Reply:
Good point. I'd still recommend removing the `${{ ... }}` bit, since we don't need string interpolation here.

Reply:
Done.